src/chttpd/src/chttpd_util.erl

```diff
@@ -95,3 +96,20 @@ handle_legacy_config(OriginalHeaders, DefaultHeaderValue) ->
         false ->
             OriginalHeaders
     end.
+
+get_dbs_info_shards(DbName) ->
+    Timeout = fabric_util:request_timeout(),
+    IsolatedFun = fun() -> fabric:get_db_info(DbName) end,
+    try fabric_util:isolate(IsolatedFun, Timeout) of
+        {ok, DbInfo} ->
+            {info, {DbInfo}};
+        {error, Error} ->
+            ErrMsg = "~p : could not get _dbs_info for Db:~p Error:~p",
```

Comment:
Logging here is probably helpful if we're papering over and returning `not_found` but wanted to also know the real reason for the failure. If we return the actual error perhaps we don't need the logging.